server/src/test/java/org/elasticsearch/index/IndexingSlowLogTests.java

```diff
@@ -272,12 +272,12 @@ public void testSlowLogParsedDocumentPrinterSourceToLog() throws IOException {
             () -> IndexingSlowLogMessage.of(index, doc, 10, true, 3));
         assertThat(e, hasToString(containsString("_failed_to_convert_[Unrecognized token 'invalid':"
             + " was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')\\n"
-            + " at [Source: (org.elasticsearch.common.bytes.AbstractBytesReference$MarkSupportingStreamInputWrapper)")));
+            + " at [Source: ")));
```

Comment:
This now throws with a slightly different text because we read straight from the `byte[]`